src/main/java/core/basesyntax/service/OperationService.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.Operation;
+
+public interface OperationService {
+    Operation getOperation(String operationType);
```

Comment:
add an applyOperation() method and all your operation have to implement this interface and override applyOperation() method or better to create OperationHandler interface with this method